bigbluebutton-html5/imports/ui/components/actions-bar/styles.scss

```diff
@@ -23,10 +27,30 @@
   flex: 0;
 }
 
-.center {
+.center,
+.nonVoicePresenter,
+.voicePresenter,
+.nonVoiceUser,
+.voiceUser {
   flex: 1;
 }
 
+.nonVoicePresenter {
+  margin-left: -11.6em;
```

Comment:
they are the values used to center the div containing the buttons on the action bar